pages/sync/bitbucket.en.mdx

```diff
@@ -0,0 +1,99 @@
+# Bitbucket sync setup guide
+[Bitbucket](https://bitbucket.org/product) is a Git-based source code repository hosting service popular among teams using Atlassian tools. 
+
+You can use the Tokens Studio plugin native integration with Bitbucket to sync your design tokens to a repository of your choice. 
+
+We support bi-directional sync, meaning you can use the plugin to: 
+- `push` JSON files full of design tokens to Bitbucket
+- `pull` the tokens stored in Bitbucket into any Figma file
+
+This means the design tokens living in code are the source of truth for our design decisions, which can be shared between design and development teams. 
+
+This doc outlines how to set up a Bitbucket repository and add it as a `Sync provider` in the plugin. 
+## How it works
+- Set up a `repository` and `personal access token` in Bitbucket.
```

Comment:
```suggestion
- Set up a `repository` and an `app password` in Bitbucket.
```

Reply:
Thank you! This was a copy paste error. good catch